src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -688,6 +689,27 @@ emit_sum_vector (MonoCompile *cfg, MonoType *vector_type, MonoTypeEnum element_t
 		return ins;
 	}
 }
+
+static MonoInst*
+emit_sum_sqrt_vector_2_3_4 (MonoCompile *cfg, MonoClass *klass, MonoInst *arg) {
+	MonoInst *sum = emit_simd_ins (cfg, klass, OP_ARM64_XADDV, arg->dreg, -1);
+	sum->inst_c0 = INTRINS_AARCH64_ADV_SIMD_FADDV;
+	sum->inst_c1 = MONO_TYPE_R4;
```

Comment:
Worth noting that Arm has typically pushed us away from using `FADDV` as it does not perform well on some hardware.

Rather instead they had us use a sequence of `FADDP` (`AddPairwise`) instructions which tend to have better perf/throughput: https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/gentree.cpp#L25190-L25210

Reply:
https://github.com/dotnet/runtime/issues/99749